Utils/secrets.py

```diff
@@ -0,0 +1,109 @@
+""" fetch secrets by key from appropriate source according to environment """
+
+from enum import Enum
+import os
+
+
+TEAM_PREFIXES = {"HIVE", "BST"}
+PROJECT_IDS = {
+    "DEV": "amazing-city-422521-t1",
+    "ANA": "ana",
```

Comment:
This should just be an environment, no? I think the Hive dev project is the analytics project?